webui-user.bat

```diff
@@ -3,6 +3,6 @@
 set PYTHON=
 set GIT=
 set VENV_DIR=
-set COMMANDLINE_ARGS=
+set COMMANDLINE_ARGS= --autolaunch --xformers
```

Comment:
Should these parameters be in this PR? They alter default startup behaviour when this PR is specifically for torch cuda updates